### PR TITLE
Changed Javadoc comments to XML comments.

### PR DESCRIPTION
**Pull request by dlemstra**
This PR changes the Javadoc comments to XML comments to make sure that comments are visibile in intellisense.

### CODE REVIEW

**Comment by slorello89** [comment]
Hi @dlemstra - thanks for this PR (been on my todo list for a while) I'll need to check to see if it's something we can support as the javadocs are actually what we use to build a lot of our documentation on the developer website.

@jeffswartz - Would this break everything?

**Review by jeffswartz — APPROVED** [review]

**Comment by jeffswartz** [comment]
> Hi @dlemstra - thanks for this PR (been on my todo list for a while) I'll need to check to see if it's something we can support as the javadocs are actually what we use to build a lot of our documentation on the developer website.
> 
> @jeffswartz - Would this break everything?

Should be good. We have some processing scripts for the Windows SDK reference that work with .NET style code comments. I may need to fiddle with that to parse the generated content and get it into the developer center.

File: OpenTok/OpenTok.cs [hunk 1]
```diff
@@ -8,47 +8,51 @@
...
-    * To create a new OpenTok object, call the OpenTok() constructor with your OpenTok API key
-    * and the API secret for your <a href="https://tokbox.com/account">OpenTok project</a>.
-    * Do not publicly share your API secret. You will use it with the OpenTok constructor
-    * (only on your web server) to create OpenTok sessions.
-    */
+    /// <summary>
+    /// Contains methods for creating OpenTok sessions, generating tokens, and working with archives.
+    /// <para>
+    /// To create a new OpenTok object, call the OpenTok() constructor with your OpenTok API key
+    /// and the API secret for your <see href="https://tokbox.com/account">OpenTok project</see>.
```
Comment (jeffswartz) [inline OpenTok/OpenTok.cs:15]: These external links do not work (at least with Doxygen generating the docs). Can you please change all `see` links to external URLs to use `a` tags (throughout)?

```suggestion
    /// and the API secret for your <a href="https://tokbox.com/account">OpenTok project</a>.
```

**Review by jeffswartz — CHANGES_REQUESTED** [review]

File: OpenTok/OpenTok.cs (hunk 1 above)
Comment (dlemstra) [reply OpenTok/OpenTok.cs:15]: Did you confirm that `a` works? `<see href="">` appears to be the official syntax (https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/xmldoc/see#remarks) but it looks like this is not supported by Doxygen (http://doxygen.nl/manual/xmlcmds.html).

Comment (slorello89) [reply OpenTok/OpenTok.cs:15]: It looks like Visual Studio supports both - @jeffswartz - does doxygen support this as well?

Comment (jeffswartz) [reply OpenTok/OpenTok.cs:15]: Yes, the `a` tag works in Doxygen. And it does not support the `<see href >` syntax.

Comment (dlemstra) [reply OpenTok/OpenTok.cs:15]: Thanks @jeffswartz. Will update the links later today.

**Comment by dlemstra** [comment]
@slorello89 I always rebase my PR's. You might want to ask if someone is okay with you merging into their branch next time. No biggie but something to consider in the future.

The links have been changed back to `<a/>`. Please let me know if I need to make additional changes.

**Review by slorello89 — APPROVED** [review]
Hi @dlemstra - sorry I pressed the GitHub `update branch` button in the PR to make sure that everything was building correctly (the windows-latest image change and removed some of the .NET Core versions the tests were running against so I needed to update the action).

Pulled everything into Visual Studio and it looks good to me. @jeffswartz - if you could just run this through Doxygen to make sure everything looks good I think we're good to merge.

**Review by jeffswartz — APPROVED** [review]

**Comment by slorello89** [comment]
Hi @dlemstra - thanks so much for this - it's a great improvement!

**Pull request merged**